infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Action.java

```diff
@@ -0,0 +1,31 @@
+package org.corfudb.infrastructure.orchestrator;
+
+import org.corfudb.runtime.CorfuRuntime;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)